File: Plugins/Android/ThirdSDK.androidlib/src/main/java/com/bridge/common/listener/IShareListener.java
package com.bridge.common.listener;

/**
 * 分享监听
 */
public interface IShareListener {
    /**
     * 分享成功
     */
    void onSuccess();

    /**
     * 用户取消分享
     */
    void onCancel();

    /**
     * 分享失败
     * @param errCode 错误码
     * @param errMsg 错误信息
     */
    void onError(int errCode, String errMsg);
}

File: Scripts/Listener/IShareListener.cs

// *******************************************
// Company Name:	深圳市晴天互娱科技有限公司
//
// File Name:		IShareListener.cs
//
// Author Name:		Bridge
//
// Create Time:		2024/09/03 10:42:31
// *******************************************

namespace Bridge.Common
{
	/// <summary>
	/// 
	/// </summary>
	public interface IShareListener
	{
		public void OnSuccess();

		public void OnCancel();

		public void OnError(int errCode, string errMsg);
	}
}